scripts/create-update.ps1

```diff
@@ -0,0 +1,92 @@
+<#
```

Comment:
Does this script require running from Administrator? If so we should comment it here. It would be even better if we could test for those permissions and fail with an error message if necessary.